docs/_pages/installation.md

```diff
@@ -118,6 +118,11 @@ It contains ROS drivers not included in the official ROS packages.
     sudo apt install jderobot
     ```
 
+5. Source the environment variables:
+    ```bash 
+    source ~/.bashrc    
+    ```
+
```

Comment:
```bash
    echo "source /opt/jderobot/setup.bash" >> ~/.bashrc
    source ~/.bashrc
    ```

Reply:
After making an installation, the packages are not added to the bashrc automatically, and thus adding them to the file is required